### PR TITLE
Jon blog

### PR DESCRIPTION
**Pull request by jonpepler**
Needs checking!

### CODE REVIEW

**Comment by mybits** [comment]
Hi @jon1467, there are repetitions in some commits, do you mind squash them for the better readability? :)

https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

File: source/assets/stylesheets/styles/_blog_content.scss [hunk 1]
```diff
@@ -106,6 +106,13 @@
     display: block;
     margin: 0 auto 20px;
     max-width: 100%;
+
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_blog_content.scss]: 🙀

File: source/blog/2016-07-22-brighton-ruby-conference-2016.html.markdown [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+[Brighton Ruby](http://brightonruby.com/) is unsuprisingly a conference about Ruby based in Brighton, and organised by [@AndyCroll](https://twitter.com/andycroll). Being quite new to both Ruby and to Rails, a Ruby conference seemed like the perfect place to pickup a few tricks of the trade and see what's next for the language and community.
+
+![](/assets/images/blog/Ruby_Team.png)
+*The Ubxd Ruby Conf team, from left to right: [Neil](https://unboxed.co/people#neil-van-beinum), [Jon](https://unboxed.co/people#jon-pepler), [Murray](https://unboxed.co/people#murray-steele), [Chris](https://unboxed.co/people#chris-holmes), [Chris II](https://unboxed.co/people#chris-carter), [Cale](https://unboxed.co/people#cale-tilford), [Ben](https://unboxed.co/people#ben-wong), and [Tom](https://unboxed.co/people#tom-sabin)*
+
+# Talks Worth Talking About
+## The Point of Objects / [John Cinnamond](https://twitter.com/jcinnamond)
+<!-- - Familiar with OOP, but this talk did a great job of reminding us how we got there and why we should appriciate it more
+- Great interactive demos
+- Interesting history -->
```
Comment (tomsabin) [inline source/blog/2016-07-22-brighton-ruby-conference-2016.html.markdown]: Do we need these comments or are they leftover notes?

File: source/blog/2016-07-22-brighton-ruby-conference-2016.html.markdown [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+>*"If procedural programming is so great, why don't I marry it?"*
+
+John talked Object-Orientated Programming as "the art of deffered caring", emphasising that OOP is structured in a way that lets you focus solely on the logic required for a specific task, delegating out required but not strictly relevant logic for later.
+
+This isn't just a concept for the average developer to consider, but paramount to the ease of use of a language. Most modern programming languages are not domain specifc, i.e. the developer and even the high level language should not have to consider the machine that code is running on, as lower level aspects handle machine-specific translations in compilation. John suggested that we have lost out on an even greater computer revolution due to developers caring too much about machine specific solutions.
+
+![](/assets/images/blog/Manchester_Mark_1.jpg)
```
Comment (tomsabin) [inline source/blog/2016-07-22-brighton-ruby-conference-2016.html.markdown]: We should add alternative text to our images to describe what the image is portraying, for users that may have images disabled or reading the article on a screen reader. As a reminder, the syntax is `alt text`

**Comment by tomsabin** [comment]
A few technical nitpicks but otherwise the blog post is a great read 👍

**Comment by tomsabin** [comment]
LGTM :shipit:

**Pull request merged**